.github/workflows/build-and-test.yaml

```diff
@@ -36,12 +36,6 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
 
-      - name: Install Protoc
-        uses: arduino/setup-protoc@v1
```

Comment:
@kezhenxu94 Could we use a third part plugin now?

Reply:
> @kezhenxu94 Could we use a third part plugin now?

This kind of actions are blocked, unless we get enough reasons to persuade the infra team to unblock it.

Reply:
![image](https://user-images.githubusercontent.com/31562192/103273919-62509800-49fb-11eb-9147-ad4fe77569c5.png)
@hanahmily  i use shell to install protoc